templates/zerver/emails/missed_message.html

```diff
@@ -34,12 +35,13 @@
         </div>
         {% endfor %}
     </div>
+    {% endif %}
 
 
     <p>
         <a href="{{ realm_uri }}">Click here to log in to Zulip and view your new messages.</a>
-        {% if reply_to_zulip  %}
```

Comment:
I wonder if it makes sense to change the definition of reply_to_zulip in the python code that calls the template, instead of adding two ifs here?

Reply:
Yeah, we can change it to python code. Done.